stoqs/loaders/CANON/loadCANON_september2016.py

```diff
@@ -176,13 +176,73 @@ def find_urls(base, search_str):
 cl.l_662_startDatetime = startdate
 cl.l_662_endDatetime = enddate
 
-# NPS_29
-#cl.nps29_base = 'http://legacy.cencoos.org/thredds/dodsC/gliders/Line66/'
-#cl.nps29_files = [ 'OS_Glider_NPS_G29_20140930_TS.nc' ]
+# NPS_29 ## not in this campaign
+cl.nps29_base = 'http://legacy.cencoos.org/thredds/dodsC/gliders/Line66/'
+cl.nps29_files = [ 'OS_Glider_NPS_Glider_29_20120524_TS.nc' ]
 cl.nps29_parms = ['TEMP', 'PSAL']
 cl.nps29_startDatetime = startdate
 cl.nps29_endDatetime = enddate
 
+# SG_621 ## KISS glider from Caltech/JPL
+cl.sg621_base = cl.dodsBase + 'CANON/2016_Sep/Platforms/Gliders/Seaglider/'
+cl.sg621_files = [ 
```

Comment:
Can be more concise with a list comprehension:

`cl.sg621_files = ['p621{:04d}'.format(i) for i in range(48,95)]`